src/sdk_version.rs

```diff
@@ -0,0 +1,87 @@
+use std::fmt;
+use serde::{Serialize, Serializer};
+
+
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum SDKVersion {
```

Comment:
I think it makes more sense to have an `ApiNumber`/`SDKNumber` enum, for example, with names such as `Api26` and so on, and then convert it to string using `as_str()`:

```Rust
impl SDKNumber {
    pub fn as_str(&self) -> &str {
        match *self {
            SDKNumber::Api26 => "Oreo",
            //...
        }
    }
}
```